modules/nw-dns-forward.adoc

```diff
@@ -71,14 +62,63 @@ spec:
 <7> Optional. You can use it to provide upstream resolvers.
 <8> You can specify two types of `upstreams` - `SystemResolvConf` and `Network`. `SystemResolvConf` configures the upstream to use `/etc/resolv.conf` and `Network` defines a `Networkresolver`. You can specify one or both.
 <9> If the specified type is `Network`, you must provide an IP address. `address` must be a valid IPv4 or IPv6 address.
-<10> If the specified type is `Network`, you can optionally provide a port. `port` must be between 1 and 65535.
+<10> If the specified type is `Network`, you can optionally provide a port. `port` must be between 1 and 65535. If you do not specify a port for the upstream, by default port 853 is tried.
```

Comment:
Suggestion to specify [what`port` is](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references):
```suggestion
<10> If the specified type is `Network`, you can optionally provide a port. The `port` field must have a value between `1` and `65535`. If you do not specify a port for the upstream, by default port 853 is tried.
```
since the numbers mentioned are potential values for that field, my thought is that they need to be wrapped in backticks as well. But feel free to disregard if not 🙂

Reply:
Great catch!